src/EOS.hpp

```diff
@@ -35,6 +36,7 @@ template <typename problem_t> struct EOS_Traits {
 	static constexpr double cs_isothermal = NAN; // only used when gamma = 1
 	static constexpr double mean_molecular_weight = NAN;
 	static constexpr double boltzmann_constant = boltzmann_constant_cgs;
+	static constexpr double hydrogen_mass_code_units = hydrogen_mass_cgs;
```

Comment:
We should not use this anymore. We always need the atomic mass unit in code units.